src/plugin/utc/index.js

```diff
@@ -1,5 +1,26 @@
 import { MILLISECONDS_A_MINUTE, MIN } from '../../constant'
 
+export const REGEX_VALID_OFFSET_FORMAT = /[+-]\d\d(?::?\d\d)?/g
+export const REGEX_OFFSET_HOURS_MINUTES_FORMAT = /([+-]|\d\d)/g
+
+function offsetFromString(value = '') {
+  const offset = value.match(REGEX_VALID_OFFSET_FORMAT)
```

Comment:
can we skip these three lines, and put everything to REGEX_OFFSET_HOURS_MINUTES_FORMAT regex?

Reply:
I am not sure I understood you well.
Can you provide more info on the changes you would like regarding this code block?

Do you want the empty line removed and have the `offsetFromString` method just below the `REGEX_OFFSET_HOURS_MINUTES_FORMAT`, with no blank space in between?

Reply:
it seems REGEX_VALID_OFFSET_FORMAT and REGEX_OFFSET_HOURS_MINUTES_FORMAT is doing the same thing?

Reply:
that make sense, thanks